test/Ocelot.AcceptanceTests/MapRequestTests.cs

```diff
@@ -0,0 +1,245 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Ocelot.Configuration.File;
+
+namespace Ocelot.AcceptanceTests
+{
+    public class MapRequestTests : IDisposable
+    {
+        private readonly ServiceHandler _serviceHandler;
+        private readonly Steps _steps;
+
+        private class ChunkedContent : HttpContent
+        {
+            private readonly string[] _chunks;
+
+            public ChunkedContent(params string[] chunks)
+            {
+                _chunks = chunks;
+            }
+
+            protected override async Task SerializeToStreamAsync(Stream stream, TransportContext context)
+            {
+                foreach (var chunk in _chunks)
+                {
+                    var bytes = Encoding.Default.GetBytes(chunk);
+                    await stream.WriteAsync(bytes, 0, bytes.Length);
+                }
+            }
+
+            protected override bool TryComputeLength(out long length)
+            {
+                length = -1;
+                return false;
+            }
+        }
+
+        public MapRequestTests()
+        {
+            _serviceHandler = new ServiceHandler();
+            _steps = new Steps();
+        }
+        
+        [Fact]
+        public void should_map_request_without_content()
+        {
+            var port = PortFinder.GetRandomPort();
+
+            var configuration = new FileConfiguration
+            {
+                Routes = new List<FileRoute>
+                {
+                    new()
+                    {
+                        DownstreamPathTemplate = "/",
+                        DownstreamScheme = "http",
+                        DownstreamHostAndPorts = new List<FileHostAndPort>
+                        {
+                            new()
+                            {
+                                Host = "localhost",
+                                Port = port,
+                            },
+                        },
+                        UpstreamPathTemplate = "/",
+                        UpstreamHttpMethod = new List<string> { "Get" },
+                    },
+                },
+            };
```

Comment:
Why not to define setup helpers?
- `GivenRoute()`
- `GivenConfiguration()`

These helpers will allow you to reduce lines number in the file in times!

Reply:
Fixed in https://github.com/alexreinert/Ocelot/commit/d077a01cd71d151c9f6c79295e1c4a07479b1760